### DISCUSSION TITLE
I Need Help

### DISCUSSION THREAD

**Discussion by naveed-gung** [SoCuul/SCInsta#89] [General]
Hi, I'm having trouble installing and running the SCInsta. I'm new to dealing with .deb files, and I'd appreciate some help.

- Device: Jailbroken iPhone X, iOS 16.7.10.
- I downloaded the .deb file from your GitHub and installed it using Filza, but I'm unsure how to inject or activate the tweak. 
- I've already tried respringing and rebooting after installation, but I don't see any new features in Instagram. 
- Is there something else I should do, or am I missing a step?

Any help or guidance would be appreciated! Thank you in advance.

**Comment by SoCuul** [comment]
Hey! Sorry for the late reply, just saw this now. Try holding down on the instagram logo on the feed page and see if the SCInsta menu pops up.

This logo here:
<img width="111" alt="image" src="https://github.com/user-attachments/assets/75541ff8-4325-4659-b58a-4467104619ee">
